README.md

```diff
@@ -0,0 +1,197 @@
+# EasyClinic API App
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍